packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -212,6 +228,8 @@ async function assertCannotCreateDb(
 
 async function assertCannotCreateCollection(
   browser: CompassBrowser,
+  // TODO(COMPASS-8002): take into account connectionName
```

Comment:
we should remove these todos now, right?

Reply:
The problem is still that the assumption is there that this is the only connection with this database name and collection name. I've changed the interface but internally it isn't robust against there being multiple connections open.